tests/test_jurisdiction_bulk_update.py

```diff
@@ -0,0 +1,64 @@
+from util.jurisdiction_bulk_update import bulk_update_jurisdictions
+from models import Election, Jurisdiction, JurisdictionAdministration, Organization, User
+import arlo_server
+import pytest
+import uuid
+
+
+@pytest.fixture
+def db():
+    with arlo_server.app.app_context():
+        arlo_server.init_db()
+
+    yield arlo_server.db
+
+    arlo_server.db.session.commit()
+    arlo_server.db.drop_all()
```

Comment:
I wonder if it makes more sense to do a `.drop_all()` and `.create_all()` at the beginning of every test... just to make sure the DB always starts in the right state? Or does `init_db()` do that?

Reply:
I was following the pattern from `test_app.py`, but I could do that instead.

Reply:
Ah, your call :)

Reply:
Changed.